pynucastro/networks/rate_collection.py

```diff
@@ -108,15 +108,18 @@ def get_nuclei(self):
         """return a list of Nuclei objects that make up this composition"""
         return list(self.X)
 
+    def len(self):
```

Comment:
This should be `__len__`, so we can do `len(composition)` rather than `composition.len()`